app.py

```diff
@@ -259,11 +259,11 @@ def discard_inspection(inspection_id):  # pragma: no cover
     return jsonify(error="Not yet implemented!"), HTTPStatus.SERVICE_UNAVAILABLE
 
 
-@app.route("/inspections", methods=["GET"])
+@app.route("/inspections/<inspection_id>", methods=["GET"])
```

Comment:
As is, this seems to break `GET /inspections`. Please run `test_get_inspection` to confirm.

I suggest a separate route.